### PR TITLE
EOD terminating multiline strings was indented, which is not allowed in PHP8

### PR DESCRIPTION
**Pull request by lostcarpark**

### CODE REVIEW

File: webpages/reports/pubsbio.php [hunk 1]
```diff
@@ -4,12 +4,13 @@
 $report['name'] = 'Pubs - Participant Bio and pubname';
 $report['multi'] = 'true';
 $report['output_filename'] = 'pubs_bios.csv';
-$report['description'] = 'Show the id, pubsname and bio for each participant who is on at least one scheduled, public session.';
-$report['categories'] = array(
+$report['description'] =
+    'Show the id, pubsname and bio for each participant who is on at least one scheduled, public session.';
```
Comment (LVerhulst4321) [inline webpages/reports/pubsbio.php:8]: Please put this back on the previous line. That's how all the other reports are done and I would like a consistent look.

File: webpages/reports/pubsbio.php [hunk 2]
```diff
@@ -4,12 +4,13 @@
...
+$report['categories'] = [
```
Comment (LVerhulst4321) [inline webpages/reports/pubsbio.php:9]: Why was this changed from the array() syntax to the [] syntax? Please change it back for consistency.

**Review by LVerhulst4321 — CHANGES_REQUESTED** [review]

File: webpages/reports/pubsbio.php (hunk 1 above)
Comment (lostcarpark) [reply webpages/reports/pubsbio.php]: Okay.

File: webpages/reports/pubsbio.php (hunk 2 above)
Comment (lostcarpark) [reply webpages/reports/pubsbio.php]: Okay, have changed it back. Though note that the $report['queries'] already used the newer [] syntax.

**Comment by LVerhulst4321** [comment]
Thanks!
